runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerConfig.java

```diff
@@ -115,7 +127,8 @@ default XpPanelLabel onScreenDisplayModeBottom()
 		position = 7,
 		keyName = "xpPanelLabel1",
 		name = "Top-left XP info label",
-		description = "Configures the information displayed in the top-left of XP info box"
+		description = "Configures the information displayed in the top-left of XP info box",
+		section = overlaySection
```

Comment:
The last four options (`xpPanelLabel[1234]`) marked here **are not part of the overlay**. These are the labels of the skill boxes displayed in the side panel.